modules/nw-cluster-mtu-change-about.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-network-mtu.adoc
+
+[id="nw-cluster-mtu-change-about_{context}"]
+= About the cluster MTU
+
+During installation the maximum transmission unit (MTU) for the cluster network is detected automatically based on the MTU of the primary network interface of nodes in the cluster. You do not normally need to override the detected MTU.
+
+You might want to change the MTU of the cluster network for several reasons:
+
+- The MTU detected during cluster installation is not correct for your infrastructure
+- Your cluster infrastructure now requires a different MTU, such as from the addition of nodes that need a different MTU for optimal performance
+
+[id="service-interruption-considerations_{context}"]
+== Service interruption considerations
+
+When you initiate an MTU change on your cluster the following effects might impact service availability:
+
+- At least two rolling reboots are required to complete the migration to a new MTU. During this time, some nodes are not available as they restart.
+
+// https://github.com/openshift/enhancements/pull/926/files#diff-166e452788d3711a8dfa7254d867f156c7b7c659f172e633acbf6852504ae358R180
+- Specific applications deployed to the cluster with shorter timeout intervals than the absolute TCP timeout interval might experience disruption during the change.
+
+// https://github.com/openshift/enhancements/pull/926/files#diff-166e452788d3711a8dfa7254d867f156c7b7c659f172e633acbf6852504ae358R190
+- Different endpoints in the cluster might use different MTUs during the migration. As a result it is possible that some network traffic is dropped until the MTU change is finalized.
+
+[id="mtu-value-selection_{context}"]
+== MTU value selection
+
+When planning your MTU migration there are two related but distinct MTU values to consider.
+
+* *Hardware MTU*: This MTU value is set based on the specifics of your network infrastructure.
+* *Cluster network MTU*: This MTU value is always less than your hardware MTU to account for the cluster network overlay overhead. The specific overhead is determined by your cluster network provider:
+** *OVN-Kubernetes*: `100` bytes
```

Comment:
I think it would be good to add a statement that only `OVN-Kubernetes` and `OpenShift SDN` are supported for MTU migration